cpp/ps0003/ps0003_Solution.cpp

```diff
@@ -1,40 +1,21 @@
-
-  /* help your brother check if the number is even or odd
-        - if the number is even, print "Number is even"
-        - if the number is odd, print "Number is odd"
-
-        - ex:
-          => 909
-          <= Number is odd
-
-        - another ex:
-          => 20000
-          <= Number is odd
-    */
-
-
 #include<iostream>
 
 int main() {
 
-  
-
-
-
+    // prompting user for number 2/2
     int number;
-
     std::cin >> number;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number